snowflake/pkg/snowflake/snowflake.go

```diff
@@ -101,3 +105,36 @@ func (c *client) DropWarehouse(ctx context.Context, name string) error {
 	_, err := c.db.ExecContext(ctx, query)
 	return err
 }
+
+func (c *client) UseDatabase(ctx context.Context, name string) error {
+	query := fmt.Sprintf("USE DATABASE %s", name)
+	c.logger.V(2).Info("Snowflake query", "query", query)
+	_, err := c.db.ExecContext(ctx, query)
+	return err
+}
+
+func (c *client) UseSchema(ctx context.Context, name string) error {
+	query := fmt.Sprintf("USE SCHEMA %s", name)
+	c.logger.Info("Snowflake query", "query", query)
```

Comment:
the verbosity values for the logs don't seem consistent (see `V(2)` above)

Reply:
Thanks, addressed.